scalafmt-tests/src/test/resources/test/Enum.stat

```diff
@@ -35,17 +32,47 @@ case Uranus extends Planet(8.686e+25, 2.5559e7)
 case Neptune extends Planet(1.024e+26, 2.4746e7)
 }
 >>>
-enum Planet (mass: Double, radius: Double) {
-  private final val G = 6.67300e-11
-  def surfaceGravity = G * mass / (radius * radius)
-  def surfaceWeight(otherMass: Double) = otherMass * surfaceGravity
-
-  case Mercury extends Planet(3.303e+23, 2.4397e6)
```

Comment:
is `case` within `enum` parsed using `KwCase` token? is the tree parsed as `Case` or something else? i only see handling of the parent `enum`, but not the `case` trees inside.

Reply:
There is an `EnumCase` token that I was not aware of at the very beginning.